CarPark/models/CarPark/Components/CarparkControl/CarparkControl/PaymentMachine/InstanceStateMachine/InstanceStateMachine.xtuml

```diff
@@ -14,7 +14,7 @@ INSERT INTO SM_EVTDI
 	"c7a393ee-03dc-44d0-a142-8335736b4fbd",
 	'TicketNumber',
 	'',
-	"ba5eda7a-def5-0000-0000-000000000002",
+	"ba5eda7a-def5-0000-0000-000000000003",
```

Comment:
Fixing same data rule violation. Here I changed int to real which should not affect the behavior in the transition.

Reply:
Actually, this is not a violation of the relaxed same-data rule required to support transition activities.  The entry activity does not access any event parameters.  Each transition activity accesses the parameter carried by the event triggering the transition.  Therefore, I submit that this change should not be necessary, and therefore we have another Ciera issue with the support of transition activities.